packages/backend/src/clients/cube/transformer.ts

```diff
@@ -65,7 +65,8 @@ const getSemanticLayerTimeGranularity = (
     }
 };
 
-const getCubeTimeDimensionGranularity = (
+// FIXME: we should not export from here. maybe common?
+export const getCubeTimeDimensionGranularity = (
```

Comment:
I don't think we can export from common since `TimeDimensionGranularity` is not available there, it comes from a package installed in backend

Reply:
ok, im gonna remove the comment